skbio/stats/tests/test_composition.py

```diff
@@ -1305,5 +1306,144 @@ def test_pairwise_vlr(self):
         self.assertAlmostEqual(output, 0.2857382286903922)
 
 
+class TestDirMultTTest(TestCase):
+    def setUp(self):
+        np.random.seed(0)
+        # Create sample data for testing
+        self.data = {
+            'feature1': [5, 8, 12, 15, 20],
+            'feature2': [3, 6, 9, 12, 15],
+            'feature3': [10, 15, 20, 25, 30],
+        }
+        self.table = pd.DataFrame(self.data)
+        self.grouping = pd.Series(['Group1', 'Group1', 'Group2', 'Group2', 'Group2'])
+        self.treatment = 'Group2'
+        self.reference = 'Group1'
+
+        d = 50
+        n = 200
+        self.depth = depth = 1000
+        p1 = np.random.lognormal(0, 1, size=d) * 10
+        p2 = np.random.lognormal(0.01, 1, size=d) * 10
+        self.p1, self.p2 = p1 / p1.sum(), p2 / p2.sum()
+        self.data2 = np.vstack(
+            (
+                [np.random.multinomial(depth, self.p1) for _ in range(n)],
```

Comment:
May consider swapping those with `rng` too. Not important though, if you are not testing the generated values.